pkg/config/security/security.go

```diff
@@ -181,3 +182,11 @@ func validateMapKey(key string) error {
 	}
 	return fmt.Errorf("bad key (%s): should have format a[b]", key)
 }
+
+// validCipherSuites contains a list of all ciphers supported in Gateway.server.tls.cipherSuites
+// TODO: DO-NOT-MERGE: extract full supported cipher suite list
+var validCipherSuites = sets.NewSet("ECDHE-ECDSA-AES128-SHA")
+
+func IsValidCipherSuite(cs string) bool {
+	return validCipherSuites.Contains(cs)
```

Comment:
Can you add that if the cs doesn't start with alphabet, log but let it pass the validation?
The idea is that these list is supposed to be a operator(e.g. exclude blabla) and operand.
I think we the validation is to figure out the obvious error but not block advanced users.

Reply:
Done